snapshot/README.md

```diff
@@ -0,0 +1,139 @@
+--- 
+layout: pattern
+title: Snapshot
+folder: snapshot
+permalink: /patterns/snapshot/
```

Comment:
This table is not formatted correctly when viewing the markdown file. Also, why do you think this design pattern needs the Gang of Four tag?

Reply:
Thanks for pointing that out, this and the wikipedia parts are from when I was under the assumption that Snapshot was Memento.